DataFormats/Phase2L1Taus/interface/L1HPSPFTau.h

```diff
@@ -0,0 +1,156 @@
+#ifndef DataFormats_Phase2L1Taus_L1HPSPFTau_H
```

Comment:
what is the difference between this and HPSPFTau in DataFormats/L1TParticleFlow/interface/

Can we not just use that class with approprate modifications